src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,136 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
     @Override
     public void add(T value) {
+        Node<T> newNode = new Node<>(null, value, tail);
```

Comment:
When creating a new Node, pass all the fields to the Node constructor. The first parameter should be the previous node, in this case, 'tail', and the next node should be 'null' since it's the last one.